application/data_access/odp_summaries/conformance.py

```diff
@@ -134,11 +133,11 @@ def get_odp_conformance_summary(dataset_types, cohorts):
     ]
 
     # Filter out fields not in spec
-    column_field_df["matching_field"] = column_field_df.replace({'"', ""}).apply(
+    column_field_df["mapping_field"] = column_field_df.replace({'"', ""}).apply(
         lambda row: [
             field
             for field in (
-                row["matching_field"].split(",") if row["matching_field"] else ""
```

Comment:
i've just seen your digital-land-builder PR, i think this should be changed to `row["mapping_field"].split(";")` (and the same for the non mapping field) given that we're now concatenating the fields with `;`, i'm not sure though